### ISSUE TITLE
Can't edit vouchers or codes

### ISSUE THREAD

**Issue by CMeldgaard** [verbb/gift-voucher#103]
### Describe the bug

Vouchers and code are no longer clickable
![image](https://user-images.githubusercontent.com/14794642/202395087-b0c71c22-eb41-439a-8d2b-4d4faf541fa0.png)
![image](https://user-images.githubusercontent.com/14794642/202395141-1de72455-db6b-492f-9831-60c7c99ba0a9.png)

### Steps to reproduce

1. Create a code or vocuher
2. Go to the element index
3. Vouchers and codes are no longer clickable

### Craft CMS version

4.3.2.1

### Plugin version

3.0.4

**Comment by engram-design** [comment]
I believe this was a a Craft 4.3.2 change. Fixed for the next release. To get the fix early, change your `verbb/gift-voucher` requirement in `composer.json` to:

```
"require": {
  "verbb/gift-voucher": "dev-craft-4 as 3.0.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [3.0.5](https://github.com/verbb/gift-voucher/releases/tag/3.0.5)
